pytket/qir/conversion/conversion.py

```diff
@@ -341,7 +344,8 @@ def _rebase_op_to_gateset(self, op: OpType, args: List) -> Optional[Circuit]:
             params = op.params
             circuit = Circuit(self.circuit.n_qubits, self.circuit.n_bits)
             circuit.add_gate(optype, params, args)
-            self.rebase_to_gateset.apply(circuit)
+            if self.rebase_to_gateset.apply(circuit):
```

Comment:
Same here.

Reply:
done in @cqc-melf
[add gateset predicate](https://github.com/CQCL/pytket-qir/pull/42/commits/833d1194d7fd7ffb2443ab8306137e62cb6d4f0e)
[833d119](https://github.com/CQCL/pytket-qir/pull/42/commits/833d1194d7fd7ffb2443ab8306137e62cb6d4f0e)